insta_reels_publishing_api_sample/index.js

```diff
@@ -33,6 +33,7 @@ const SCOPES = [
     "instagram_content_publish"
 ];
 const STRINGIFIED_SCOPES = SCOPES.join("%2c");
+const RATE_LIMIT_ALLOWED_QUOTA = 25; // 25 published posts every 24 hours
```

Comment:
I think you can get quota_total from the API response. No need to hardcode this. Also this will break if the quota changes in the future

Reply:
`rateLimitResponse.data.data[0].config.quota_total`

Reply:
Oh yes. Good catch !! Will update this.